packages/netlify-cms-widget-list/src/ListControl.js

```diff
@@ -277,7 +277,7 @@ export default class ListControl extends React.Component {
           allowAdd={field.get('allow_add', true)}
           onAdd={this.handleAdd}
           heading={`${items.size} ${listLabel}`}
-          label={listLabel}
+          label={labelSingular.toLowerCase() || listLabel}
```

Comment:
We don't actually need to set the fallback to `listLabel` here, since `labelSingular` already has the fallback builtin to its definition above. We should just be able to do `labelSingular.toLowerCase()`.